starter-code/app.js

```diff
@@ -1,5 +1,61 @@
 const express = require('express');
 const app = express();
+const expressLayouts = require('express-ejs-layouts');
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+// -- setup
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+// -- routes
+
+app.get('/random', (req, res, next) => {
+        // Retrieve a random chuck joke
+        next("SDFAFSDFSDFSDF");
+    client.getRandomJoke()
+    .then((response) => {
+        res.render('index',response);
+    }).catch((err) => {
+        next(err);
+    });
+});
+  
+app.get('/categories', (req, res, next) => {
+//    console.log(req.query);
+    client.getJokeCategories()
+    .then((response)=>  {
+        const data = {
```

Comment:
Here you are doing what you needed to do in the first get, wrap the information you want to pass to the render in an objetc. Good job.